tests/test_upstream_issue.py

```diff
@@ -15,12 +15,9 @@ class TestUpstreamIssue(unittest.TestCase):
     def setUp(self):
         self.mock_config = {
             'sync2jira': {
-                'default_github_project_fields': {
-                    'storypoints': ('Estimate', 'customfield_12310243')
-                },
                 'map': {
                     'github': {
-                        'org/repo': {'sync': ['issue']},
+                        'org/repo': {'sync': ['issue'], 'github_project_fields': {}},
                     },
```

Comment:
So, by default, we're testing with an empty `'github_project_fields'`, shouldn't we have tests where it has entries in it?

Reply:
I have added tests for both the story points and priority.